### PR TITLE
Get content images url from data-src attribute

### PR DESCRIPTION
**Pull request by orballo**
fixes https://github.com/frontity/saturn-theme/issues/171

### CODE REVIEW

File: src/shared/processors/low/image.js [hunk 1]
```diff
@@ -52,7 +52,11 @@ export default {
     let src = null;
 
     // Get src attribute from different cases.
-    if (props.src && typeof props.src === 'string') {
+    if (props['data-src'] && typeof props['data-src'] === 'string') {
+      if (props.src.startsWith('/'))
+        src = `${settings.generalSite.url}${props.src}`;
```
Comment (luisherranz) [inline src/shared/processors/low/image.js]: Shouldn't this be:
```js
if (props['data-src'].startsWith('/'))
        src = `${settings.generalSite.url}${props['data-src']}`;
```
?

If that's the case, add a test as that part of the code is not covered.

File: src/shared/processors/low/image.js [hunk 2]
```diff
@@ -68,7 +72,19 @@ export default {
...
+        .map(s => {
+          const trimmed = s.trim();
+
+          if (trimmed.startsWith('/'))
+            return `${settings.generalSite.url}${trimmed}`;
+
+          return trimmed;
+        })
+        .join(', ');
+    } else if (props.srcSet && typeof props.srcSet === 'string') {
```
Comment (luisherranz) [inline src/shared/processors/low/image.js]: Why are you repeating code? (same with `props.src` above)

**Comment by orballo** [comment]
@luisherranz ready for review.

**Pull request merged**